contracts/interfaces/IDCAStrategies.sol

```diff
@@ -397,6 +394,22 @@ interface IDCAStrategiesPositionsHandler {
     uint256[] positions
   );
 
+  /**
+   * @notice Emitted when a user withdraws all swapped tokens from a position
+   * @param withdrawer The address of the user that executed the withdraw
+   * @param recipient The address of the user that will receive the withdrawn tokens
+   * @param positionId The id of the position that was affected
+   * @param underlyingsPositionId The ids of the underlying positions affected
+   * @param tokenAmounts The amounts withdrawn and respective tokens
+   */
+  event Withdrew(
+    address indexed withdrawer,
+    address indexed recipient,
+    uint256 positionId,
+    uint256[] underlyingsPositionId,
```

Comment:
I think we can avoid this. It costs gas to emit data, and this information should already be available off-chain, since we already emitted the underlying position ids in `deposit`